yet-another-docker-plugin/src/main/resources/com/github/kostyasha/yad/commons/DockerCreateContainer/help-linksString.html

```diff
@@ -1 +1,5 @@
-Add link to another container.
+<div>
```

Comment:
div wrapper is not needed, AFAIR my research jenkins wraps in div in any case

Reply:
Ok.